app/src/main/java/woowacourse/movie/setting/AlarmReceiver.kt

```diff
@@ -0,0 +1,32 @@
+package woowacourse.movie.setting
+
+import android.content.BroadcastReceiver
+import android.content.Context
+import android.content.Intent
+import woowacourse.movie.SettingPreference
+import woowacourse.movie.main.MainActivity.Companion.SETTING_PREFERENCE_KEY
+import woowacourse.movie.mapper.movie.mapToDomain
+import woowacourse.movie.seat.view.SeatSelectionActivity.Companion.BOOKING_MOVIE_KEY
+import woowacourse.movie.ticket.model.BookingMovieModel
+import woowacourse.movie.utils.getParcelableCompat
+
+class AlarmReceiver : BroadcastReceiver() {
+    override fun onReceive(context: Context, intent: Intent) {
+        val settingPreference = SettingPreference(context)
+        if (intent.action == ALARM_CODE && settingPreference.getBoolean(SETTING_PREFERENCE_KEY)) {
+            val notificationBuilder = NotificationBuilder(context)
+            notificationBuilder.createNotificationChannel()
+
+            val bookingMovie = intent.getParcelableCompat<BookingMovieModel>(BOOKING_MOVIE_KEY)
+                ?.mapToDomain()
+            if (bookingMovie != null) {
+                notificationBuilder.createNotificationBuilder(bookingMovie)
+            }
+        }
+    }
```

Comment:
from: https://github.com/woowacourse/android-movie-theater/pull/46#discussion_r1187495307
```suggestion
class AlarmReceiver : BroadcastReceiver() {
    val settingPreference = SettingPreference(context)
    override fun onReceive(context: Context, intent: Intent) {
        if (intent.action == ALARM_CODE && settingPreference.getBoolean(SETTING_PREFERENCE_KEY)) {
            val notificationBuilder = NotificationBuilder(context)
            notificationBuilder.createNotificationChannel()

            val bookingMovie = intent.getParcelableCompat<BookingMovieModel>(BOOKING_MOVIE_KEY)
                ?.mapToDomain()
            if (bookingMovie != null) {
                notificationBuilder.createNotificationBuilder(bookingMovie)
            }
        }
    }
```
위와 같이 바꿔보면 어떨까요?
> AlarmReceiver에서는 onReceive될때마다 인스턴스화하는 것이 아닌, 클래스 프로퍼티로 두어 활용할 수 있을 것 같아요.